src/basic/unit-def.h

```diff
@@ -146,6 +146,7 @@ typedef enum ServiceState {
         SERVICE_FAILED,
         SERVICE_DEAD_BEFORE_AUTO_RESTART,
         SERVICE_FAILED_BEFORE_AUTO_RESTART,
+        SERVICE_RESOURCES_PINNED,  /* Like SERVICE_DEAD, but with pinned resources */
```

Comment:
maybe call it SERVICE_DEAD_RESOURCES_PINNED to make it clear that it's a dead state?